docusaurus/docs/1_operate/3_configs/3_supplier_staking_config.md

```diff
@@ -142,7 +142,9 @@ account.
 
 ### `owner_address`
 
-_`Required`_
+| Scenario | Requirement  |
```

Comment:
Can you add your table to the docs (this or other) as well.

The table has enough permutations that we'll need to refer people back to it.